SnakeGame/SnakeGame/Game.cs

```diff
@@ -0,0 +1,184 @@
+﻿namespace SnakeGame
+{
+    public class Game
+    {
+        enum Directions
+        {
+            Right,
+            Left,
+            Up,
+            Down
+        }
+
+        public bool HasEnded = false;
+
+        private Dictionary<string, Directions> movementDirections 
```

Comment:
Do we need a dictionary? Could we define the enum other way so we don't need this dictionary?